x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/action/InferTrainedModelDeploymentAction.java

```diff
@@ -215,7 +234,7 @@ public void writeTo(StreamOutput out) throws IOException {
                 out.writeBoolean(skipQueue);
             }
             if (out.getVersion().onOrAfter(Version.V_8_7_0)) {
-                out.writeOptionalString(textInput);
+                out.writeOptionalStringCollection(textInput);
```

Comment:
Shouldn't we have an `else` where we handle previous versions.

Reply:
There is nothing else to write